blackbox/docs/appendices/release-notes/3.0.0.rst

```diff
@@ -0,0 +1,223 @@
+.. _version_3.0.0:
+
+=============
+Version 3.0.0
+=============
+
+Released on 2018/05/16.
+
+Changelog
+=========
+
+Breaking Changes
+----------------
+
+- The ``region`` setting for ``CREATE REPOSITORY`` has been removed. It is
+  automatically "guessed" but can still be manually specified by using the
+  ``endpoint`` setting.
+
+- Renamed EC2 ``cloud.aws.*`` settings to ``discovery.ec2.*`` (eg.
+  ``cloud.aws.access_key`` is now ``discovery.ec2.access_key``)
+
+- The columns ``number_of_shards``, ``number_of_replicas``, and
+  ``self_referencing_column_name`` in ``information_schema.tables`` changed to
+  return ``NULL`` for non-sharded tables.
+
+- Adapted queries in the Admin UI to be compatible with CrateDB 3.0 and
+  greater.
+
+- For HTTP authentication, support was dropped for the ``X-User`` header, used
+  to provide a username, which has been deprecated in ``2.3.0.`` in favour of
+  the standard HTTP ``Authorization`` header.
+
+- Dropped support for tables that have been created with CrateDB prior to
+  version 2.0. Tables which require upgrading are indicated in the cluster
+  checks, including visually shown in the Admin UI, if running the latest 2.2
+  or 2.3 release. The upgrade of tables needs to happen before updating CrateDB
+  to this version. This can be done by exporting the data with ``COPY TO`` and
+  importing it into a new table with ``COPY FROM``.  Alternatively you can use
+  ``INSERT`` with query.
+
+- Data paths as defined in ``path.data`` must not contain the cluster name as a
+  folder. Data paths which are not compatible with this version are indicated
+  in the node checks, including visually shown in the Admin UI, if running the
+  latest 2.2 or 2.3 release.
+
+- The ``bootstrap.seccomp`` setting that controls system call filters has been
+  replaced in favour of ``bootstrap.system_call_filter``.
+
+- The ``error_trace`` GET parameter of the HTTP endpoint only allows ``true``
+  and ``false`` in lower case. Other values are not allowed any more and will
+  result in a parsing exception.
+
+- Store level throttling settings (``indices.store.throttle.*``) have been
+  removed.
+
+- The gateway recovery table setting ``recovery.initial_shards`` has been
+  removed. Nodes will recover their unassigned local primary shards
+  immediatelly after restart.
+
+- The ``_node`` column on ``sys.shards`` and ``sys.operations`` has been
+  renamed to ``node``, is now visible by default and has been trimmed to only
+  include ``node['id']`` and ``node['name']``. In order to get all information
+  a join query can be used with ``sys.nodes``.
+
+- ``discovery.type`` setting has been removed. To enable EC2 discovery
```

Comment:
There are three points about removed settings, but all are phrased slightly different. Maybe we could unify the wording, e.g.

* The store level throttling settings ``indices.store.throttle.*`` have been removed.
* The gateway table setting `recovery.initial_shards` has been removed. ...
* The discovery setting `discovery.type` has been removed. ...